tools/IoTEdgeDevOps/TestDashboard/dashboards/Home-1585707775101.json

```diff
@@ -2032,7 +2035,7 @@
           "colors": [
             "rgba(245, 54, 54, 0.9)",
             "rgba(237, 129, 40, 0.89)",
-            "rgba(50, 172, 45, 0.97)"
+            "#73BF69"
```

Comment:
why changed to color hex?

Reply:
It is generated, I was poking around and messed with the color wheel.